roles/core/templates/sdcore-5g-values.yaml

```diff
@@ -314,18 +314,19 @@ omec-user-plane:
         enabled: false    # SRIOV is disabled by default
       hugepage:
         enabled: false    # Should be enabled if DPDK is enabled
-      cniPlugin: macvlan  # Can be any other plugin. Dictates how IP address are assigned
```

Comment:
I think the `ipam` parameter also has to go inside each interface.

Reply:
done